src/app/Library/CrudPanel/Traits/Columns.php

```diff
@@ -372,6 +372,12 @@ public function makeSureColumnHasNeededAttributes($column)
             $column['relation_type'] = $this->inferRelationTypeFromRelationship($column);
         }
 
+        // if there's a model defined, but no attribute
+        // guess an attribute using the identifiableAttribute functionality in CrudTrait
+        if (isset($column['model']) && ! isset($column['attribute']) && method_exists($column['model'], 'identifiableAttribute')) {
+            $column['attribute'] = call_user_func([(new $column['model']()), 'identifiableAttribute']);
```

Comment:
@maurohmartinez I know it's not your fault, this works as expected, but I learned after I coded that in Fields that `call_user_func` is less efficient than calling the method directly on the instance `$instance->identifiableAttribute()`  and it's not really necessary in this case (and in the case of the Fields too) I would rather we not use it here.

I think we can reorganize some of this methods that are used in Fields and Columns and are the same, if we had a function `makeSureElementHasAttribute(array $element) {}` we would be using it here and in fields too! We need to give this some thinking before work on it, just letting you know what I think about this for future reference. 

For now, let's just call the function directly from the model instance as it should have been 👍